includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -580,6 +580,8 @@ private function maybe_add_lightbox_attributes( $attributes, $node ) {
 	private function is_gif_url( $url ) {
 		$ext  = self::$anim_extension;
 		$path = wp_parse_url( $url, PHP_URL_PATH );
+		// 检查$path是否为null，若是则赋值为空字符串
+		$path = $path?? '';
```

Comment:
```suggestion
		if ( null === $path ) {
			return false;
		}
```

Reply:
Thank you very much for taking the time to review my PR and providing such valuable suggestions. I really appreciate.